freud/locality.pyx

```diff
@@ -1120,17 +1120,28 @@ cdef class Voronoi(_Compute):
     def __dealloc__(self):
         del self.thisptr
 
-    def compute(self, system):
+    def compute(self, system, radii=None):
         R"""Compute Voronoi diagram.
 
         Args:
             system:
                 Any object that is a valid argument to
                 :class:`freud.locality.NeighborQuery.from_system`.
+            radii:
+                A list of :class:`numpy.ndarray` defining radii of points in
+                the system. (Default value = :code:`None`, meaning unbiased
+                Voronoi diagram.)
```

Comment:
Follow this example of how to provide an array of values for each point: https://github.com/glotzerlab/freud/blob/0e695ea11daf5385c546bfa4688c2009064aee93/freud/density.pyx#L84-L86
```suggestion
            radii ((:math:`N_{points}`) :class:`numpy.ndarray`):
                An array of radii for each point in the system. If provided,
                the power diagram (also called the radical Voronoi
                tessellation) will be computed (Default value = :code:`None`,
                which gives the Voronoi diagram).
```